Update step-down.mdx (#19329)
hashicorp/vault: website/content/docs/commands/operator/step-down.mdx

There are no flags beyond the [standard set of flags](/vault/docs/commands)
included on all commands.

> ## Policy Requirements

Reviewer: ```suggestion
## Policy requirements
```

Style correction: use sentence case for headings

> In order to access this feature you will need a policy with (at minimum) the following:

Reviewer: ```suggestion
You must belong to a policy with `update` and `sudo` permissions to use the
`vault operator step-down` command:
 
```

Style correction: write in active voice

> ```

Reviewer: ````suggestion
```hcl
````

Style correction: label example code with language
